integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomainNoLB.java

```diff
@@ -0,0 +1,870 @@
+    String domainUid = domain.getSpec().getDomainUid();
+    String domainNamespace = domain.getMetadata().getNamespace();
+
+    String managedServerPodNamePrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+
+    int numberOfServers = 3;
+    logger.info("Scaling cluster {0} of domain {1} in namespace {2} to {3} servers.",
+        clusterName, domainUid, domainNamespace, numberOfServers);
+    List<String> managedServersBeforeScale = listManagedServersBeforeScale(replicaCount);
+    scaleAndVerifyCluster(clusterName, domainUid, domainNamespace, managedServerPodNamePrefix,
+        replicaCount, numberOfServers, null, managedServersBeforeScale);
+
+    // then scale cluster back to 2 servers
+    logger.info("Scaling cluster {0} of domain {1} in namespace {2} from {3} servers to {4} servers.",
+        clusterName, domainUid, domainNamespace, numberOfServers, replicaCount);
+    managedServersBeforeScale = listManagedServersBeforeScale(numberOfServers);
+    scaleAndVerifyCluster(clusterName, domainUid, domainNamespace, managedServerPodNamePrefix,
+        numberOfServers, replicaCount, null, managedServersBeforeScale);
+  }
+
+  /**
+   * Verify admin console login using admin node port.
+   * Skip the test for slim images due to unavailability of console application
+   * @param domain oracle.weblogic.domain.Domain object
+   */
+  @ParameterizedTest
+  @DisplayName("Test admin console login using admin node port")
+  @MethodSource("domainProvider")
+  void testAdminConsoleLoginUsingAdminNodePort(Domain domain) {
+
+    assumeFalse(WEBLOGIC_SLIM, "Skipping the Console Test for slim image");
+
+    assertDomainNotNull(domain);
+    String domainUid = domain.getSpec().getDomainUid();
+    String domainNamespace = domain.getMetadata().getNamespace();
+
+    String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+    logger.info("Getting node port for default channel");
+    int serviceNodePort = assertDoesNotThrow(() -> getServiceNodePort(
+        domainNamespace, getExternalServicePodName(adminServerPodName), "default"),
+        "Getting admin server node port failed");
+
+    logger.info("Validating WebLogic admin server access by login to console");
+    boolean loginSuccessful = assertDoesNotThrow(() ->
+        adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT),
+        "Access to admin server node port failed");
+    assertTrue(loginSuccessful, "Console login validation failed");
+  }
+
+  /**
+   * Test rolling restart for a mii domain.
+   * Make sure pods are restarted only once.
+   * Verify all pods are terminated and restarted only once
+   * Rolling restart triggered by changing:
+   * imagePullPolicy: IfNotPresent --> imagePullPolicy: Never
+   * Verify domain changed event is logged.
+   */
+  @Test
+  @DisplayName("Verify server pods are restarted only once by changing the imagePullPolicy in mii domain")
+  void testClustersRollingRestart() {
```

Comment:
Modify the tests name to testMiiClusterRollingRestart() since it is specific to Mii domain only.

Reply:
fixed